sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-generation.service.ts

```diff
@@ -497,4 +508,38 @@ export class ECertGenerationService {
       }
     }
   }
+
+  /**
+   * Calculate tuition remittance effective amount.
+   * @param disbursements all disbursements that are part of one e-Cert.
+   */
+  private calculateTuitionRemittanceEffectiveAmount(
+    disbursements: ECertDisbursementSchedule[],
+  ) {
+    for (const disbursement of disbursements) {
+      const award: Award[] = disbursement.disbursementValues?.map(
```

Comment:
If `disbursement.disbursementValues` are not present on the DB they will not be part of the query result, hence the null check can be avoided.